src/Compilers/Extension/source.extension.vsixmanifest

```diff
@@ -9,7 +9,7 @@
     <InstallationTarget Version="[15.0,]" Id="Microsoft.VisualStudio.Pro" />
   </Installation>
   <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.7.2,)" />
+    <!-- <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.7.2,)" /> -->
```

Comment:
><!-- <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.7.2,)" /> --> [](start = 4, length = 128)

remove?

Reply:
Forgot this is still here. This is a temporary workaround for VSIX installation bug. Will remove once we have a VS builds with this bug fixed.